### PR TITLE
Add a setup page documenting steps to build and deploy your own binaries for the tool

### PR DESCRIPTION
**Pull request by nmeierpolys**
This was my first time doing anything with Go so I'm absolutely open to any suggestions or changes to the setup recommendations there.

### CODE REVIEW

**Comment by cristim** [comment]
Looking great, thanks for this contribution.

File: SETUP.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+removing your dependency on the author's version, and allowing any customizations
+and improvements your organization needs. You'll need to set up a local environment
+to run Go, compile the binaries locally, upload them to an S3 bucket in your AWS
+account, and update the CloudFormation stack to use those new binaries.
+
+## Compiling the binaries locally ##
+
+1. Install Go via the binaries [here](https://golang.org/dl/) or Homebrew:
+
+   `brew install go`
```
Comment (xlr-8) [inline SETUP.md]: Mac dependant, but if people come here, I assume they already have Go installed or know how to use a package manager.

File: SETUP.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+   dependencies.
+
+6. Navigate to the root of the AutoSpotting repository:
+
+   `cd src/github.com/cristim/autospotting`
+
+7. Try building and running the tests to make sure everything pulled in correctly. More 
+   details on the available directives below.
+
+   `make install`
```
Comment (xlr-8) [inline SETUP.md]: This rule is outdated:
```bash
$ make install
make: *** No rule to make target 'install'.  Stop.```

File: SETUP.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+   the tool is running your own custom-built version. If so, add a line like this to the 
+   `autospotting.go` file's `main()` function:
+
+   `fmt.Println("Running <my organization name> binaries")`
+
+## Using your own binaries in AWS ##
+
+1. Set up an S3 bucket in AWS that will host the binaries
+
+2. The Makefile contains a `BUCKET_NAME` variable that tells it where to upload new
```
Comment (xlr-8) [inline SETUP.md]: Outdated as well, it's the build_helper.sh scripts that does so now

File: SETUP.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+   binaries. Edit the Makefile and change it from the default `cloudprowess` to the name
```
Comment (xlr-8) [inline SETUP.md]: I think the default has been changed to `cloudprowess-private` - mentioned in various places afterwards as well

File: SETUP.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+
+1. Set up an S3 bucket in AWS that will host the binaries
+
+2. The Makefile contains a `BUCKET_NAME` variable that tells it where to upload new
+   binaries. Edit the Makefile and change it from the default `cloudprowess` to the name 
+   of your S3 bucket.
+
+3. Build and upload your binaries to the S3 bucket
+
+   `make release`
```
Comment (xlr-8) [inline SETUP.md]: Outdated

File: SETUP.md [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+   * Verifies that the necessary dependencies are installed
+   * Runs `go build` to compile the project for deployment
+   * Retrieves the `instances.json` file
+   * Bundles the binaries into a `lambda.zip` file under `build/s3/dv`
+* **release**:
+   * Runs the `install` directives
+   * Uploads the binaries from `build/s3` to the specified S3 bucket
+* **test**:
+   * Runs `go build` to compile the project locally
+   * Runs the tool against a JSON file of test data
```
Comment (xlr-8) [inline SETUP.md]: Outdated as well

**Pull request merged**